Epoch 1 of 2 starting
Traceback (most recent call last):
  File "/home/bmn/projects/ml_system/project/test_trainer.py", line 62, in <module>
    train_loss, val_loss = trainer.fit(epochs=config.epochs)
                           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/bmn/projects/ml_system/project/trainer/trainer.py", line 61, in fit
    self._call_callbacks('on_epoch_end', epoch, train_loss, val_loss, y_train_pred, y_train, y_val_pred, y_val)
  File "/home/bmn/projects/ml_system/project/trainer/trainer.py", line 72, in _call_callbacks
    method(self, *args)
  File "/home/bmn/projects/ml_system/project/trainer/callbacks/base.py", line 24, in on_epoch_end
    for name, metric in self.metrics.items():
                        ^^^^^^^^^^^^^^^^^^
AttributeError: 'NoneType' object has no attribute 'items'
